modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase

```diff
@@ -215,7 +215,7 @@ definition {
 			ObjectNotifications.addNewNotificationTemplate(
 				bcc = "test3@liferay.com",
 				cc = "test2@liferay.com",
-				description = "Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
+				description = "Lorem Ipsum is simply dummy text of the printing and typesetting industry.Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
```

Comment:
Why is this change part of 913e3f7f1777cfbae83bf83165d45b875aaa189e? It's not a logical part of the changes you're making